src/types/index.ts

```diff
@@ -233,25 +235,27 @@ export interface IPermission {
 export type NotificationStatus = 'CREATED' | 'PEEKED' | 'READ';
 
 export interface INotification {
-  wallet?: boolean
-  senderName?: string
-  receiverName?: string
-  createdAt: string
-  entityId?: string
-  entityType?: string
-  id?: number
-  path?: RawLocation
-  receiver?: string
-  sender?: string
-  sourceId?: string
-  sourceType?: string
-  status: NotificationStatus
-  text?: string | TranslateResult
-  type: string
-  updatedAt?: string
-  isSeedBackup?: boolean
-  buttonLabel?: TranslateResult,
-  title?: TranslateResult,
+  wallet?: boolean;
+  senderName?: string;
+  receiverName?: string;
+  createdAt: string;
+  entityId?: string;
+  entityType?: string;
+  id?: number | string;
+  path?: RawLocation;
+  receiver?: string;
+  sender?: string;
+  sourceId?: string;
+  sourceType?: string;
+  status: NotificationStatus;
+  text?: string | TranslateResult;
+  type: string;
+  updatedAt?: string;
+  isSeedBackup?: boolean;
+  isIncomingTransaction?: boolean;
```

Comment:
I suggest to rename it to `incomingTransactionExists` or `hasIncomingTransaction`.